pkg/pb/payments.proto

```diff
@@ -12,14 +13,17 @@ service Payments {
     rpc Calculate(CalculateRequest) returns (CalculateResponse);
     // AdjustPrices sets the prices paid by a satellite for data at rest and bandwidth
     rpc AdjustPrices(AdjustPricesRequest) returns (AdjustPricesResponse);
+        // GenerateCSV creates a csv file for payment purposes
+    rpc GenerateCSV(GenerateCSVRequest) returns (GenerateCSVResponse);
+    //TODO REMOVE
```

Comment:
Should this be removed now? Or sometime in the future?

Reply:
will remove in ticket https://storjlabs.atlassian.net/browse/V3-1084 :)